data_contribution_2_data.md

```diff
@@ -0,0 +1,907 @@
+    `)
+  } else send.clear()
+} catch(e) { }
+</script>
+
+### data/endpoints
+
+The endpoints directory holds the final results created as part of a research analysis. Often, these are files created to support papers or grants, and other dissemination. Further sub-directories can be added to organize data, if necessary.
+
+**Within an [Arcus Scientific Lab](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/Arcus_Labs_Orientation/main/arcus_orientation.md#1)**
+
+- Data in this directory is managed by the study team.
+- Data in this directory will be saved if the project is archived in Arcus.
+
+**Endpoints** data is different depending on the type of research. Please select what type(s) of data you would like more information about, you can select both:
+
+- [ ] omics data
+- [ ] clinical data
+<script output="data_type_endpoints">"@input"</script>
+
+<script modify="false">
+try {
+  let data_type = @input(`data_type_endpoints`)
+
+  if(data_type[0]) {
+    send.liascript(`
+    ## Omics Data
+
+![Omics data/endpoints](media/project_template/endpoints_omics.gif)<!-- style = "max-height: 500px;" -->
+
+This directory is for files created at the end of a bioinformatics workflow, like gvcf and vcf files.
+
+- **gvcf** files contain variant information, describing genomic regions with no variants for a single sample. They are used to compare variant calls across samples to make vcf files. We collect them to enable the easy construction of larger cohorts. We collect the compressed version of a gvcf file (g.vcf.gz) which can be made with a lossless compression utility tool like [gzip](https://www.gzip.org). All gvcf files should include an index file, .tbi
+- **vcf** files usually contain multiple samples, and are the starting point for most reserach project's analysis. They are not appropriate for constructing cohorts from multiple projects because they are missing necessary information contained in the gvcf files. We collect the compressed version of a vcf file (vcf.gz), which can be made with a lossless compression utility tool like gzip[gzip](https://www.gzip.org). The variant calls in these files might differ from those in a gvcf file because they are made by considering information from all samples. A researcher would want to use these files when considering each project separately. All vcf files should include an index file, .tbi
+
+    `)
+  } else send.clear()
+} catch(e) { }
+</script>
+
+<script modify="false">
+try {
+  let data_type = @input(`data_type_endpoints`)
+
+  if(data_type[1]) {
+    send.liascript(`
+    ## Clinical Data
+
+![Clinical data/endpoints](media/project_template/endpoints_clinical.gif)
```

Comment:
Similarly, relying on an image here without copying the text into alt text or supplemental text may hinder usability in some cases. I leave it to you to decide if the effort of re-doing is worth it.